pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -2870,3 +2877,47 @@ func releaseAppNetworkResources(ctx *zedrouterContext, key string,
 	}
 	publishAppNetworkStatus(ctx, status)
 }
+
+func isDNSServerChanged(new *types.DeviceNetworkStatus) bool {
```

Comment:
"new" is not an ideal name of a variable, since it is also a built-in function.

Reply:
Sure, will change.